packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/PurchaseNFTs/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { colors } from '@blockchain-com/constellation'
+import styled from 'styled-components'
+
+import {
+  Button,
+  Image,
+  Link,
+  SkeletonCircle,
+  SkeletonRectangle,
+  Text
+} from 'blockchain-info-components'
+import { Flex } from 'components/Flex'
+import FlyoutHeader from 'components/Flyout/Header'
+import { media } from 'services/styles'
+
+import { Props as OwnProps } from '..'
+
+const PurchaseNFTs: React.FC<Props> = (props) => {
+  const { close } = props
+
+  const CustomText = styled(Text)`
+    text-align: center;
+    width: 418px;
+    ${media.tablet`
+  width: 100%;
+  `}
+  `
+
+  const NumberCircle = styled.div`
+    border-radius: 50%;
+    width: 18px;
+    height: 18px;
+    padding: 4px;
+    background: ${colors.grey000};
+    text-align: center;
+  `
+
+  return (
+    <>
+      <Flex flexDirection='column' gap={16} alignItems='center'>
+        <FlyoutHeader sticky data-e2e='howDoesThisWork' mode='back' onClick={() => close()}>
+          <FormattedMessage id='copy.how-does-this-work' defaultMessage='How Does This Work' />
+        </FlyoutHeader>
+        <Flex flexDirection='column' gap={16} alignItems='center'>
+          <Image width='200px' name='nft-purchase' />
+          <Text color='grey900' size='20px' weight={600}>
+            Purchase NFTs
+          </Text>
+          <CustomText color='grey700' size='16px' weight={500}>
+            Buy an NFT using your Blockchain.com app and Walletconnect on Opensea or Rarible.
+          </CustomText>
+          <Flex flexDirection='column' alignItems='flex-start' style={{ padding: '1em' }}>
+            <Text color='grey900' size='16px' weight={600}>
+              Instructions
+            </Text>
+            <Flex alignItems='center' gap={8} style={{ padding: '1.5em 0em' }}>
+              <NumberCircle>
+                <Text color='grey600' weight={600}>
+                  1
+                </Text>
+              </NumberCircle>
+              <div>
+                <Text color='grey900' weight={600} size='16px'>
+                  Go To One Of The Marketplace’s Above
+                </Text>
+              </div>
+            </Flex>
+            <Flex alignItems='center' gap={8}>
+              <NumberCircle>
+                <Text color='grey600' weight={600}>
+                  2
+                </Text>
+              </NumberCircle>
+              <div>
+                <Text color='grey900' weight={600} size='16px'>
+                  Open Your Blockchain.com App
+                </Text>
+              </div>
+            </Flex>
+            <Flex alignItems='center' gap={8} style={{ padding: '1.5em 0em 0em' }}>
```

Comment:
Prefer to use the <Padding /> component here instead of passing a style attribute. style attribute on flex/padding is going to be removed in the new constellation components.